packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -1723,6 +1723,52 @@ describe('src/cy/commands/actions/click', () => {
           cy.get('#dom').invoke('css', 'scrollBehavior').then((scrollBehavior) => expect(scrollBehavior).to.eq('smooth'))
         })
       })
+
+      describe('retries in after hook when failures', () => {
+        it('clicks element in hook', (done) => {
+          cy.on('fail', (err) => {
+            expect(err.message).contain('expected true to be false')
+            done()
```

Comment:
I don't think you want to use `done()` in these tests, otherwise the test is going to pass even if the assertions in the `after` fail.

Reply:
@mschile If you remove the code for the fix, this test does fail in the after hook.

Reply:
Yes, but the test will also pass if you modify any of the assertions in the `after` hook since you've already informed mocha that the test is done.